src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-script-property/elsa-script-property.tsx

```diff
@@ -89,6 +127,17 @@ export class ElsaScriptProperty {
       </div>
       {fieldHint ? <p class="elsa-mt-2 elsa-text-sm elsa-text-gray-500">{fieldHint}</p> : undefined}
       <input type="hidden" name={fieldName} value={value}/>
+      {renderValidationResult()}
+      {isSchema ?
+        <div class="elsa-mt-3">
+          <a href="#"
+              onClick={e => this.onConvertToJsonSchemaClick(e)}
+              class="elsa-relative elsa-inline-flex elsa-items-center elsa-px-4 elsa-py-2 elsa-border elsa-border-gray-300 elsa-text-sm elsa-leading-5 elsa-font-medium elsa-rounded-md elsa-text-gray-700 elsa-bg-white hover:elsa-text-gray-500 focus:elsa-outline-none focus:elsa-shadow-outline-blue focus:elsa-border-blue-300 active:elsa-bg-gray-100 active:elsa-text-gray-700 elsa-transition elsa-ease-in-out elsa-duration-150">
+              Convert to Json Schema
+          </a>
+        </div>
```

Comment:
This portion should be provided by the HTTP Endpoint plugin as well, since it is the owner of the Schema property.